recipes/gametes/gametes.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+from os import access, getenv, path, X_OK
+
+
+# jar file
+JAR_NAME = 'gametes.jar'
+PKG_NAME = 'gametes'
+PKG_VERSION = '2.1'
+PKG_BUILDNUM = '0'
+
+
+default_jvm_mem_opts = ['-Xms512m', '-Xmx1g']
+
+#def real_dirname(in_path):
```

Comment:
Could you please remove commented-out code chunks, now that it's building successfully?